simulator_service/launch/simulation.launch

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" ?>
+
+<!-- 
+  This launch file brings up a series of nodes in a special namespace
+  that simulate the same robot loaded normally.
+ -->
+
+<launch>
+
+  <arg name="namespace" default="simulation"/>
+  <arg name="world_frame" default="world_frame"/>
+  
+  <group ns="$(arg namespace)">
+
+    <!-- publish the robot state (tf transforms) in a namespaced topic -->
+    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
+      <param name="tf_prefix" value="$(arg namespace)"/>
+    </node>
+
+    <!-- Connects the simulation world_frame with the real world_frame so rviz can visualize both at once -->
+    <node pkg="tf" type="static_transform_publisher" name="world_to_sim_publisher" 
+      args="0 0 0 0 0 0 
+            $(arg world_frame) $(arg namespace)/$(arg world_frame) 1000"/>
+
+    <!-- Lets the controller read from the real robot's joint names -->
+    <remap from="controller_joint_names" to="/controller_joint_names"/>
+    
+    <!-- Launches a robot simulator with associated action server -->
+    <include file="$(find industrial_robot_simulator)/launch/robot_interface_simulator.launch"/>
+
+    <node pkg="simulator_service" type="simulator_service_node" name="simulator_service" output="screen"/>
+
+  </group>
+
+</launch>
```

Comment:
suggestion: give this launch file a more descriptive name. Something like `start_simulation_server.launch` or similar. Or `load_...`.